.circleci/config.yml

```diff
@@ -420,3 +419,37 @@ workflows:
             - build
             - test
             - e2e-cli
+
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - 10.1.x
+    jobs:
+      # Linux jobs
+      - setup
+      - build:
+          requires:
+            - setup
+          filters:
```

Comment:
This filter can be dropped (completely actually).  The `docs-preview` branch doesn't exist anymore.